Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -106,6 +106,21 @@ public enum eBrowserType
         [UserConfiguredDescription("Use Browser In Private/Incognito Mode (Please use 64bit Browse with Internet Explorer ")]
         public bool BrowserPrivateMode { get; set; }
 
+        [UserConfigured]
+        [UserConfiguredDefault("false")]
+        [UserConfiguredDescription("Only for Chrome & Firefox | Set \"true\" to run the browser in background (headless mode) for faster Execution")]
+        public bool HeadlessBrowserMode { get; set; }
+
+        [UserConfigured]
+        [UserConfiguredDefault("false")]
+        [UserConfiguredDescription("Only for IE, Chrome & Firefox | Set \"true\" to Launch the Browser minimized")]
+        public bool BrowserMinimized { get; set; }
```

Comment:
seems like BrowserMinimized is relevent for all browser types (Edge as well) so no need to mention "Only for IE, Chrome,..."

Reply:
Yes, it is relevant to all but it doesn't work for 'Edge'.

Reply:
even if it is doesn't work for Edge, remove the Only for...